libraries/streaming-e2e-tests/build/run-tests.yml

```diff
@@ -1,25 +1,21 @@
-# Run Tests navigating to a React App that uses a customized DLJS
-# that has latest nightly version of `botframework-streaming`
+# Run Tests navigating to a React App that uses DLJS with the latest
+# nightly version of `botframework-streaming`
 
 steps:
   - task: DownloadPipelineArtifact@2
     displayName: Download ReactApp Artifact
     inputs:
       artifact: ReactApp
       path: $(ReactProjectDir)/build
-  
+
   - powershell: start powershell {npx serve -p 3000}
     displayName: 'npx serve -p 3000'
     workingDirectory: $(ReactProjectDir)/build
-  
-  - powershell: npm install
+
+  - script: npm install
     displayName: 'npm install tests'
-    condition: ne(variables.CacheRestored, 'true')
     workingDirectory: $(TestsDir)
 
-  - powershell: Start-Sleep -Seconds 20
-    displayName: 'Sleeping for 20 seconds to allow React App to serve...'
-  
-  - powershell: npm test
+  - script: npm test
```

Comment:
I thought the sleep timer has a reason.

Test box on VSTS/ADO is pretty slow, they are [Azure DS2v2](https://docs.microsoft.com/en-us/azure/virtual-machines/dv2-dsv2-series#dsv2-series).